lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -395,34 +408,54 @@ namespace {
     return Call.get();
   }
 
-  void ValueExtractionSynthesizer::FindAndCacheRuntimeDecls() {
+  static bool VSError(clang::Sema* Sema, clang::Expr* E, llvm::StringRef Err) {
```

Comment:
Can you teach the `Lookup` routines to not suppress the diagnostics if told? I.e. use the `DiagSetting` technique from `LookupHelper` also for `Lookup` and then pass `WithDiagnostics` here?